dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/user/UserNotification.java

```diff
@@ -109,6 +112,14 @@ public void setObjectId(Long objectId) {
         this.objectId = objectId;
     }
 
+    public String getReturnToAuthorReason() {
```

Comment:
Some comment that it is intended to use only for specific NotificationType would be beneficial.